src/Reacher.Common/Logic/EmailContentRenderer.cs

```diff
@@ -25,15 +25,15 @@ public Task<string> GetPaymentSuccessEmailBody(string reacherEmail, string reach
             MarkdownContentItems = new[] {
         $@"Hey there,
 
-We just wanted to let you know that your tip of **{cost:c}** was received, and Reacher has delivered your email to **{reacherName}**."
+We just wanted to let you know that your payment of **{cost:c}** was received, and Reacher has delivered your email to **{reacherName}**."
             },
             OtherEmail = new() { Html = html, Subject = subject, Label = "**Your Email:**", AttachmentCount = attachmentCount },
             MarkdownContentItemsAfter = new[] {
-                @$"Please note that **all tips go directly to {reacherName}**. 
+                @$"Please note that **all payments go directly to {reacherName}**. 
```

Comment:
Not sure if tip is entirely accurate... but if it is used for some sort of financial lingo reason, rather than payment, then keep it as tip. I just don't really see it as very accurate since a tip is paid without the expectation of something being provided, which is not the case in this situation.

Reply:
Thanks for the suggestions. I may have to get some legal advice on this, as I'm trying to conform to Strike's terms of service and avoid any liability that I can.